code/modules/modular_computers/computers/item/computer_files.dm

```diff
@@ -84,3 +86,22 @@
 			if(file.filename == filename)
 				return file
 	return null
+
+/**
+ * find_file_by_uid
+ *
+ * Will check all files in this computer and returns the file with the matching uid.
+ * If a computer disk is passed instead, it will check the disk over the computer.
+ */
+/obj/item/modular_computer/proc/find_file_by_uid(uid, obj/item/computer_disk/target_disk)
```

Comment:
This doesn't describe what a uid is, I know what it is generally but this is the first I'm hearing of modular computers using them?